<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEProteinScannerAppGUIClass</class>
 <widget class="QWidget" name="SEProteinScannerAppGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Voxel size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxVoxelSize">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Sliding Window Size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxWindowSize">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> Voxel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxContactDistance">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Contact Distance</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditPath"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxRotate"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>nb rotations x axis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Scan database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Predict</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinScannerAppGUIClass</receiver>
   <slot>onScan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinScannerAppGUIClass</receiver>
   <slot>onPredict()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinScannerAppGUIClass</receiver>
   <slot>onBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScan()</slot>
  <slot>onBrowse()</slot>
  <slot>onPredict()</slot>
  <slot>onRotate()</slot>
 </slots>
</ui>
